Html form desktop version

bhushan354 opened the pull request:
- Added proper html form .
- Fixed horizontal scrolling of the website.
- Used correct github flow .

Thank you ,
Bhushan.

bhushan354 committed:
HTML form desktop version

bhushan354 committed:
linters fix

usorfaitheloho commented on style.css:
```diff
@@ -1614,4 +1605,14 @@ input#Email {
+    display: flex;
+    justify-content: flex-start;
+  }
 }
```
Kindly implement these changes for different sections of your project to improve the UI of your project

**NAVBAR SECTION**

-  Please adjust the margin of the nav items so it is away from the border line just as it is in the design given 😄 

Figma design :
![image](https://github.com/bhushan354/My-Portfolio/assets/20537712/a8b91fcd-f5da-4f7f-be2d-0e93ee6d3236)

Your design :
![image](https://github.com/bhushan354/My-Portfolio/assets/20537712/065e3e38-fb4f-4a77-a5eb-8ef236c126c1)

**PROJECT SESSION**

- Kindly use the right color for the heading `My recent works` so it is similar to the design given 

Figma design :
![image](https://github.com/bhushan354/My-Portfolio/assets/20537712/7b2a42b6-8442-488c-82fc-6be395724c2a)

Your design :
![image](https://github.com/bhushan354/My-Portfolio/assets/20537712/e8956358-33a2-4628-b2fd-44e2366e3a0d)

-  Kindly add some padding in the technolies of your project section
 Figma design :
![image](https://github.com/bhushan354/My-Portfolio/assets/20537712/821e858e-4de6-4753-bca5-496d217fbe5d)

Your design :
![image](https://github.com/bhushan354/My-Portfolio/assets/20537712/40d7d99a-65aa-424d-9b13-1121b789d3ba)

**ABOUT ME **

-  Kindly rescue the margin-bottom here, its really big and can make viewing unpleasurable , please it should be similar to the figma template. Also, use the right font-type and font properties for your project , especially in this section 
-  Please use the right background-color for this section. Your design background-color is darker than what we have in the design 😄 

your design :
![image](https://github.com/bhushan354/My-Portfolio/assets/20537712/42255671-177e-4687-a193-ccc15a4241b1)

**CONTACT SECTION**

- Kindly change the background-color for this text in your design :

![image](https://github.com/bhushan354/My-Portfolio/assets/20537712/764fa7ad-3e81-4d63-9952-9ff60dc7833f)

usorfaitheloho reviewed (changes requested):
> Hi @bhushan354  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

 ![](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966) [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * All linter checks passed ✔️
> * Descriptive pull request ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
>  Kindly ensure you Update your README file `Key features`, and `Future features' to be related to the features provided in your project such as ` Add projects`, `Add about-myself` for key features

> Also, please create a folder for all your images and then add them to the folder because your   project directory looks a bit disorganized and confusing 😄  

> Lastly,  please remove this styling you added on `line 51` of `style.css` as it makes your project  to look really different from what you should have ` border: 1px solid red ` 

![image](https://github.com/bhushan354/My-Portfolio/assets/20537712/f52f0128-4e75-473e-b54d-8b16d8db8c27)
> This is how your project looks like

![image](https://github.com/bhushan354/My-Portfolio/assets/20537712/dec34b69-336c-4ff9-ad63-0221aab124a4)
 

> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

bhushan354 committed:
reviewer changes

bhushan354 committed:
background colour for about me page

bhushan354 commented:
Hello sir, thank you for reviewing my projects I have made all the changes the previous reviewer asked me to do .

Take a look :

-  Removed "border : 1px solid red" from line 51 in CSS and updated README.md file .

-  Adjusted the margin of the nav items :
![Screenshot (76)](https://github.com/bhushan354/My-Portfolio/assets/127464886/46605e43-5f5a-4fc7-b099-dd996d521cd1)

- Used right color for the heading My recent works so it is similar to the design given :
![Screenshot (77)](https://github.com/bhushan354/My-Portfolio/assets/127464886/8ca3a4cb-052a-4fb4-a3c3-ba43c345f321)

- Added some padding in the technologies of my project section :
![Screenshot (78)](https://github.com/bhushan354/My-Portfolio/assets/127464886/26b51615-f11c-44fc-80d1-ab28f57c26c7)

- Removed excessive gap in about me page and added correct background color 👍:
![Screenshot (81)](https://github.com/bhushan354/My-Portfolio/assets/127464886/5fdc9556-9bb8-412f-97ec-a3afe1444e68)

- Added proper background color and text color to contact form section :
![Screenshot (80)](https://github.com/bhushan354/My-Portfolio/assets/127464886/00512471-e033-433d-9690-27709b051e3c)

Thank you ,
Bhushan .

AbdusSattar-70 commented on style.css:
```diff
@@ -1614,4 +1623,14 @@ input#Email {
     width: 200px;
     height: 40px;
   }
+
+  .lineD {
+    margin-top: 698px;
+    width: 100%;
+  }
+
+  .subInput :nth-child(4) {
+    display: flex;
+    justify-content: flex-start;
+  }
```
- [ ] Great work🎉😍 thus far on incorporating the style changes mentioned by the [previous reviewer.](https://github.com/bhushan354/My-Portfolio/pull/4/files#r1263689373)
- [ ]  However, I feel there is still an opportunity for further improvement in terms of the overall project styling. 
- [ ] I have prepared a video for your reference, where I point out specific areas that could benefit from modifications in order to align your design with the Figma design provided.
- [ ]  By ensuring your design matches the Figma design, you'll be able to captivate the attention of future recruiters. 
- [ ] Thank you for your attention to this matter.🤔🎉🥱

https://github.com/bhushan354/My-Portfolio/assets/118761303/88051611-4140-4019-aacd-5a377c08994f

AbdusSattar-70 reviewed (changes requested):
STATUS: REQUIRED CHANGES  ♻️ 
Hi @bhushan354 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- LINTERS ARE PASSED🎉
- STRIVE TO FOLLOW REVIEWER'S SUGGESTIONS ❤️
- PROFESSIONAL README 👍 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏
![AlmostGIF](https://github.com/bhushan354/My-Portfolio/assets/118761303/a519f01e-0800-42f0-a97b-3b29066427c8)

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

bhushan354 committed:
changes

bhushan354 committed:
remove borders for all elements

bhushan354 committed:
additional touch up

bhushan354 commented:
Hello sir , thank you for reviewing my project , I have made necessary changes suggested by previous reviewer and I hope this time it will meet all requirement .

I just wanted to let you know that this is my last review so if there is any small change required , it would be of great help to me if you message me in slack about that .

Thank you for consideration ,
Bhushan

Munsa1 reviewed (approved):
Hi @bhushan354, 
![Approved 1](https://github.com/bhushan354/My-Portfolio/assets/81932478/d9ad3669-8d7c-464f-8265-24188be0d2a2)

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Mobile First Design :heavy_check_mark: 
Nice UI Design :heavy_check_mark: 
Made Requested Changes By Previous Reviewer :heavy_check_mark: 

## Optional suggestions
- [ ] Consider always capitalizing the first letter of the first word of your commit messages for more professional documentation

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

bhushan354 committed:
margin error
